.env.template

```
@@ -10,3 +10,10 @@ DATABASE_URL="postgresql://USERNAME:PASSWORD@localhost:5432/DATABASE?schema=publ
 # Docker Config
 POSTGRES_USER=
 POSTGRES_PASSWORD=
+
+# Nodemailer Config
+EMAIL_SERVICE=
+EMAIL_USER=
+EMAIL_PASS=
+EMAIL_HOST=
+EMAIL_PORT=
```

Adicionar linha em branco ao final do arquivo.
```suggestion
EMAIL_PORT=

```